packages/ui/src/components/va-input/VaInput.vue

```diff
@@ -164,6 +167,25 @@ export default defineComponent({
     } = useClearable(props, modelValue, input, computedError)
 
     const { computedValue, onInput } = useCleave(input, props, valueComputed)
+    const computedValueProxy = props.type === 'number'
+      ? computed({
+        get: () => computedValue.value as number,
+        set: (value: number) => truncInRange(value, min, max),
+      })
+      : computedValue
+
+    const truncInRange = (value: number, min: Ref<number | undefined>, max: Ref<number | undefined>) => {
+      if (props.type === 'number') {
+        if (typeof min.value !== 'undefined' && value < min.value) {
+          valueComputed.value = min.value
+          return
+        } else if (typeof max.value !== 'undefined' && value > max.value) {
+          valueComputed.value = max.value
+          return
+        }
+      }
+      valueComputed.value = value
+    }
```

Comment:
Hi, @m0ksem, I don't know why this wouldn't work. I can't see this in vue devtool, doesn't show any prop or variable.

Reply:
I think it is better to use `VaCounter` instead of `VaInput type="number"`